### PR TITLE
chore: add 3dicons resource ✨

### LINKED ISSUE #936: [Add] - 3DICONS RESOURCE 💡
### Site URL

https://3dicons.co/

### Description of the above provided site

- All 3D icons you need in one place. This is a collection of free, beautiful, trending 3D icons, that you can use in your website, app or in any project.
- Over 4000 3D icons in 4k resolution, 5 different color palettes, figma and PNG files, front view and angled perspective view.
- The best part is we can use completely free and without attribution for personal or commercial project.

### Category

Frontend

### Sub Category

icons

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by 0xabdulkhalid**
## Fixes Issue

Closes #936 

## Changes proposed
- Added '3dicons' resource for Frontend icons category at `icons.json` file

## Screenshots

image

### CODE REVIEW

**Comment by rupali-codes** [comment]
Hey, your vercel deployment is getting failed of the following errors:

image

**Comment by 0xabdulkhalid** [comment]
Hello @rupali-codes, i don't know why this is happening.

The changes looks good to me, we can try once again.

**Comment by 0xabdulkhalid** [comment]
I think now's the preview has been deployed successfully

File: database/frontend/icons.json [hunk 1]
```diff
@@ -138,5 +138,12 @@
     "url": "https://www.iconfinder.com/",
     "category": "frontend",
     "subcategory": "icons"
+  },
+  {
+    "name": "3dicons",
+    "description": "All the 3D icons you need in one place. A collection of free, beautiful, trending 3D icons that you can use in your website, app, or project.",
```
Comment (rupali-codes) [inline database/frontend/icons.json:144]:
```suggestion
    "description": "It provides a collection of free, beautiful and trending 3D icons that you can use in your projects.",
```

**Comment by 0xabdulkhalid** [comment]
Done @rupali-codes

**Review by rupali-codes — APPROVED** [review]
lgtm

**Pull request merged**